MapboxNavigation/Style.swift

```diff
@@ -227,6 +227,11 @@ open class Style: NSObject {
     public var trafficSevereColor: UIColor?
     
     /**
+     Describes the situations in which the style should be used. By default, the style will be used during the daytime.
+     */
+    public var styleType: StyleType = .daytimeStyle
```

Comment:
Sounds reasonable. You might have to add a dynamic var mapStyleURL to `NavigationMapView` and set the MGLMapViews’s styleURL in the didSet.